spec/javascripts/unit/darkswarm/services/cart_spec.js.coffee

```diff
@@ -77,7 +84,7 @@ describe 'Cart service', ->
       expect(Cart.update_enqueued).toBe(true)
 
   describe "updating the cart", ->
-    data = {variants: {}}
+    data = {variants: 1: {quantity: 1, max_quantity: null}}
```

Comment:
If we revert this change the specs all pass. This is messing with tested calls to `Cart.update()`, but the code you've changed is only affecting `Cart.adjust()`, so I think it's correct to revert this bit.

Reply:
@sauloperez

Reply:
yep, it was just leftovers from my dirty iterations

Reply:
Ok, so I'll remove this plus the meaningless change on cart.js.coffee and publish the PR